tests/new_unit_tests/Commands/Import/ImportFilesCommandTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Import;
+
+use Pantheon\Terminus\Commands\Import\ImportFilesCommand;
+use Terminus\Exceptions\TerminusException;
+use Pantheon\Terminus\UnitTests\Commands\CommandTestCase;
+use Terminus\Models\Workflow;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Import\ImportDatabaseCommand
+ */
+class ImportfilesCommandTest extends CommandTestCase
```

Comment:
While I do need to double-check the rest of the command files for this, to keep the commands uniform, this class should be `FilesCommandTest`.